web/src/engine/websites/decorators/Common.ts

```diff
@@ -521,6 +522,35 @@ export function ImageElement(includeRefererHeader = true, detectMimeType = false
     };
 }
 
+export async function FetchImageAjaxFromHTML(page: Page, priority: Priority, signal: AbortSignal, queryImage: string, detectMimeType = false): Promise<Blob> {
+    const image = await this.imageTaskPool.Add(async () => {
+        const request = new FetchRequest(page.Link.href, {
+            signal: signal
+        });
+        const realimage = (await FetchCSS<HTMLImageElement>(request, queryImage))[0].src;
+        return new Page(this, page.Parent as Chapter, new URL(DeProxify(new URL(realimage)), this.URI));
```

Comment:
The de-proxification should be consistent with the other fetch image method.
Options:
- Remove it from this method
- Add it to all other fetch image methods
- Make it optional using a parameter for all fetch image methods

Reply:
> Make it optional using a parameter for all fetch image methods.

Seems like the best option, with deproxify being false by default. This wont affect all Madara & MangaStream plugins since they deproxify by themselves.